db/migrate/20230523175242_add_category_to_counties.rb

```diff
@@ -0,0 +1,5 @@
+class AddCategoryToCounties < ActiveRecord::Migration[7.0]
+  def change
+    add_column :counties, :category, :string
```

Comment:
This is really an enum rather than a string - can we do it this way? https://blog.saeloun.com/2022/01/12/rails-7-adds-custom-enum-support-in-postgresql/#after

Reply:
It seems that we can.